### ISSUE TITLE
Blue selection bubble doesn't disappear after address selection

### ISSUE THREAD

**Issue by bitbooster** [hsarkanen/JollaOpas#17]
Another minor issue in v0.8.0 is that the blue selection 'bubble' doesn't disappear after a selection has been made.

To reproduce:
- open the app
- start typing an address in the From or To field
- let the app search for matching addresses. A blue 'bubble' with the number of options (generally 10) appears on the top and a yellow dot shows at the right of the input field
- tap the bubble and pick an address
- the dot on the right turns green to mark the address is precise, but the blue selection bubble doesn't disappear

This one I'm pretty sure it's a regression as I don't remember it happening in previous versions.

**Comment by hsarkanen** [comment]
Thanks for reporting this.

I tested with the old v0.7.5 where the old, not Digitransit API, still works in Tampere and can confirm that it worked as you descibed before. But is this a real issue that the blue selection bubble stays there? In my opionion if I select accidentally wrong suggestion from the list provided by the blue selection bubble then it is actually faster to reselect the correct one since the bubble is still available.

Partly related to this in v0.8.1 fc97f80e1146d1b98512e54a423fb5b27cfba0c6 I added removing the blue selection bubble when the text field is cleared using the virtual keyboard backspace or the `X` button at the end of the line.
